packages/relay/src/lib/clients/cache/redisCache.ts

```diff
@@ -137,6 +137,28 @@ export class RedisCache implements ICacheClient {
     // TODO: add metrics
   }
 
+  /**
+   * Stores multiple key-value pairs in the cache.
+   *
+   * @param keyValuePairs - An object where each property is a key and its value is the value to be cached.
+   * @param callingMethod - The name of the calling method.
+   * @param requestIdPrefix - Optional request ID prefix for logging.
+   * @returns {Promise<void>} A Promise that resolves when the values are cached.
+   */
+  async mSet(keyValuePairs: Record<string, any>, callingMethod: string, requestIdPrefix?: string): Promise<void> {
+    // Serialize values
+    const serializedKeyValuePairs: Record<string, string> = {};
+    for (const [key, value] of Object.entries(keyValuePairs)) {
+      serializedKeyValuePairs[key] = JSON.stringify(value);
+    }
+
+    // Perform mSet operation
+    await this.client.mSet(serializedKeyValuePairs);
+
+    // Log the operation
+    this.logger.trace(`${requestIdPrefix} caching multiple keys via ${callingMethod}`);
```

Comment:
Say how many entries are cached in the group

Reply:
✅ Done